.gitsigners

```diff
@@ -0,0 +1 @@
+hyy4x65ebz48cwge8w11pk9qcu5xj7s4ipnzjsr4foez6qf4kpxqcc ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDT/bQG+jsoZB6Sk1X3Mnt6e21VorpsTRYIv5xdKa9zG
```

Comment:
what's this?

Reply:
nothing that should be there, let me remove that!